source/documentation/set-up-developers.html.md

```diff
@@ -35,7 +35,7 @@ Subscribing allows an application access to all of the endpoints of a particular
 
 ## API access control
 
-Some of the CDS APIs (and their documentation) are only accessible to applications that have been whitelisted for that API. If you cannot subscribe to or access the documentation of a particular API on the developer hub, please check with [SDST](mailto:SDSTeam@hmrc.gsi.gov.uk) if your ‘ApplicationId’ needs to be added to the whitelist for that API.
+Some of the CDS APIs (and their documentation) are only accessible to applications that have been placed in the allowlist for that API. If you cannot subscribe to or access the documentation of a particular API on the developer hub, please check with [SDST](mailto:SDSTeam@hmrc.gsi.gov.uk) if your ‘ApplicationId’ needs to be added to the allowlist for that API.
```

Comment:
i think it should be one word for code refs but two for documentation purposes.

Reply:
hypenated in docs? Or just a space?